src/System.Net.Requests/tests/HttpWebResponseHeaderTest.cs

```diff
@@ -94,6 +94,46 @@ public void HttpContinueMethod(int StatusCode, WebHeaderCollection httpHeaders)
             });
         }
 
+        [OuterLoop]
```

Comment:
If you're using the loopback server, you probably don't need to have these tests be OuterLoop.

Reply:
I followed the pattern from other tests doing http request, no personal preference here